### PR TITLE
models: extra info addition on SessionActivity

### PR DESCRIPTION
**Pull request by hachreak**
* Adds extra information inside session activity. (addresses #52)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -22,9 +22,6 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
-[aliases]
```
Comment (jirikuncar) [inline setup.cfg]: Why do you need to change the order?

Comment (hachreak) [inline setup.cfg]: thanks to remember me.. I forgot to remove this change XD

File: setup.py [hunk 2]
```diff
@@ -97,7 +97,9 @@
     'SQLAlchemy-Utils[ipaddress]>=0.31.0',
     'cryptography>=1.3',
     'invenio-i18n>=1.0.0b2',
+    'maxminddb-geolite2==2017.404',
```
Comment (jirikuncar) [inline setup.py]: Is it necessary to pin the exact version? What if one wants to upgrade it?

File: invenio_accounts/models.py [hunk 3]
```diff
@@ -135,6 +135,21 @@ class SessionActivity(db.Model, Timestamp):
 
     user = db.relationship(User, backref='active_sessions')
 
+    country = db.Column(db.String(80), nullable=True)
```
Comment (lnielsen) [inline invenio_accounts/models.py]: Perhaps we should store just an ISO code instead so we are language neutral

**Comment by hachreak** [comment]
ping @lnielsen

File: invenio_accounts/sessions.py [hunk 4]
```diff
@@ -42,6 +45,27 @@
 _sessionstore = LocalProxy(lambda: flask.current_app.kvsession_store)
 
 
+def _get_user_country():
+    """Get user country."""
+    user_real_ip = flask.request.headers.get(
```
Comment (lnielsen) [inline invenio_accounts/sessions.py]: You can assume that the real ip address is in ``request.remote_addr``. When the request is forwarded you need to use the WSGIProxyFix middleware.

Comment (hachreak) [inline invenio_accounts/sessions.py]: how can I mock the remote address from `flask.request.remote_addr`?

**Pull request merged**